test/check-application

```diff
@@ -217,6 +221,8 @@ class TestApplication(testlib.MachineCase):
         cert_path = "/etc/pki/tls/certs/myCert.cert"
         key_path = "/etc/pki/tls/private/myKey.key"
         m.execute("selfsign-getcert request -f {0} -k {1}".format(cert_path, key_path))
+        self.addCleanup(m.execute, "getcert stop-tracking -f {} -k {}".format(cert_path, key_path))
+        self.addCleanup(m.execute, "rm {} {}".format(cert_path, key_path))
```

Comment:
We don't need this `rm` in all other tests?

Reply:
Reading from the `getcert-request` manual, by default it stores the certificates in the NSS database, with `-f` and `-k` the issued certificate/keyfile are stored in the given location and requires a cleanup.

Stop tracking the certificates seems to be good enough for subsequent tests to request a new certificate but when a certificate is created on disk, it needs to be cleaned up for other tests.